hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html

```diff
@@ -45,23 +45,24 @@ <h2>Node Status</h2>
 </div>
 <table class="table table-bordered table-striped col-md-6">
     <thead>
-        <tr>
-            <th ng-click = "columnSort('hostname')" class="nodeStausInfo"><span ng-class = "{'sorting' : (columnName != 'hostname'), 'sortasc' : (columnName == 'hostname' && !reverse),
-                                        'sortdesc':(columnName == 'hostname' && reverse)}">HostName</span></th>
-            <th ng-click = "columnSort('opstate')" class="nodeStausInfo" ><span ng-class="{'sorting' : (columnName != 'opstate'), 'sortasc' : (columnName == 'opstate' && !reverse),
-                                        'sortdesc':(columnName == 'opstate' && reverse)}">Operational State</span></th>
-            <th ng-click = "columnSort('comstate')" class="nodeStausInfo">  <span ng-class="{'sorting' : (columnName != 'comstate'), 'sortasc' : (columnName == 'comstate' && !reverse),
-                                        'sortdesc':(columnName == 'comstate' && reverse)}">Commisioned State</span> </th>
-        </tr>
+    <tr>
```

Comment:
Here again, I am not sure what is causing the indentation change, probably 4 spaces exchanged to tabs, or vice versa, but we should stick to keep 4 spaces as basic indentation as it is in the rest of the file.
Also these changes seem to be related again to fixing things with sorting, so these as well should go under a separate JIRA in a separate PR.